internal/opatranslator/opa_translator.go

```diff
@@ -86,10 +88,8 @@ func (c *OPAClient) ProcessQuery(pq *rego.PartialQueries) (bson.M, error) {
 
 func processTerm(query string) []string {
 	splitQ := strings.Split(query, ".")
-	var result []string
-	for _, term := range splitQ {
-		result = append(result, removeOpenBrace(term))
-	}
+	result := lo.Map(splitQ, removeOpenBrace)
```

Comment:
I don't know the library (I'm going to check it out asap) what occurs in the map func? Is it syntactic sugar for a for loop or does it do something more? I'm worried about the performance overhead it may introduce on this piece of code, is there any?

Reply:
i've seen the lib; the operations are the same so this should be a noop in terms of performances